x/marketmap/types/ticker.go

```diff
@@ -57,7 +57,11 @@ func (t *Ticker) ValidateBasic() error {
 		return fmt.Errorf("metadata json field is longer than maximum length of %d", MaxMetadataJSONFieldLength)
 	}
 
-	return json.IsValid([]byte(t.Metadata_JSON))
+	if err := json.IsValid([]byte(t.Metadata_JSON)); err != nil {
+		return fmt.Errorf("invalid JSON metadata; %w", err)
+	}
```

Comment:
This error was not informative enough IMO

Reply:
https://github.com/skip-mev/slinky/pull/266/commits/5ca6a1e42f0606c5f223e79aed981f6eee429c24